api/BluetoothDevice.json

```diff
@@ -633,19 +579,11 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BluetoothDevice/uuids",
           "support": {
-            "chrome": [
-              {
-                "version_added": "52",
-                "notes": "Chrome origin trials only."
-              },
-              {
-                "version_added": "45",
-                "notes": "Behind a flag. Chrome OS only."
-              }
-            ],
+            "chrome": {
+              "version_added": null
+            },
             "chrome_android": {
-              "version_added": "52",
-              "notes": "Chrome origin trials only."
+              "version_added": null
```

Comment:
`false`

Reply:
I haven't been marking it, but `false` should be applied to `webview_android` as well for all of the interface members that I have marked.

Reply:
👍 It looked to me that the top-level `BluetoothDevice` wasn't supported in WebVeiw, so I marked all the subfeatures as `false` too, on my previous pass on the data

Reply:
(Or rather, I left that data as is—I didn't modify it in this PR)